### DISCUSSION TITLE
List view image is too short.

### DISCUSSION THREAD

**Discussion by DheyohDev** [awcodes/filament-curator#507] [Q&A] [answered]
Hi everyone, I'm having trouble with image list view is to short . Can anyone help me out, Like the image below.?

![Grid view filament curator](https://github.com/user-attachments/assets/9bae337d-e3cb-4a41-a0c2-4765ba6caf30)

**Comment by awcodes** [comment, marked as answer]
Make sure you follow the instructions in the read about setting up the plugin with a custom theme.

**Reply by DheyohDev** [reply]
okay it works, I don't use croppes.js and don't use custom theme panel it turns out it has an effect
